util/translate_fight.py

```diff
@@ -183,36 +185,18 @@ def main(args):
             continue
         timeline_replace.append({
             'locale': lang,
-            'replaceText': ability_replace[lang],
-            'replaceSync': mob_replace[lang],
+            'replaceSync': dict(sorted(mob_replace[lang].items(), reverse=True)),
```

Comment:
This seems reasonable to me.  We can put everything in the list reversed to start as a way to help folks writing translations.  Things can always be reordered as desired after running this script.